clippy_lints/src/extra_unused_type_parameters.rs

```diff
@@ -38,7 +41,30 @@ declare_clippy_lint! {
     complexity,
     "unused type parameters in function definitions"
 }
-declare_lint_pass!(ExtraUnusedTypeParameters => [EXTRA_UNUSED_TYPE_PARAMETERS]);
+
+#[derive(Default)]
```

Comment:
Shouldn't derive `Default` here.

Reply:
Not exactly sure why I added that. I think the example lint I looked at that used a config option (`upper_case_acronyms`) had that, so I figured it was required.